pages/common/pnpm-audit.md

```diff
@@ -0,0 +1,37 @@
+# pnpm audit
+
+> Scan project dependencies.
+> Check for known security issues with the installed packages.
+> More information: <https://pnpm.io/cli/audit>.
+
+- Identify vulnerabilities in the project:
+
+`pnpm audit`
+
+- Automatically fix vulnerabilities:
+
+`pnpm audit fix`
+
+- Generate a security report in JSON format:
+
+`pnpm audit --json > {{path/to/audit-report.json}}`
+
+- Audit only [D]ev dependencies:
+
+`pnpm audit -D`
+
+- Audit only [P]roduction dependencies:
+
+`pnpm audit -P`
```

Comment:
```suggestion
`pnpm audit --dev`

- Audit only [P]roduction dependencies:

`pnpm audit --prod`
```
Since we highlight the short option as mnemonics, maybe we could use the long option here.